xbmc/video/VideoDatabase.cpp

```diff
@@ -8558,7 +8564,27 @@ bool CVideoDatabase::GetMoviesByWhere(const std::string& strBaseDir, const Filte
         {
           // Display the name of the movie for a collection of movie assets rather than "Assets"
           if (!items.HasProperty("customtitle"))
-            items.SetProperty("customtitle", movie.GetTitle());
+          {
+            int msgId;
```

Comment:
I know not actually needed here, but for future safety: defualt init?

Reply:
I raise you a lambda to make it const int :-)